crates/red_knot_python_semantic/src/types/infer.rs

```diff
@@ -854,6 +854,9 @@ impl<'db> TypeInferenceBuilder<'db> {
             "isinstance" if definition.is_builtin_definition(self.db) => {
                 Some(KnownFunction::IsInstance)
             }
+            "issubclass" if definition.is_builtin_definition(self.db) => {
+                Some(KnownFunction::IsSubclass)
+            }
```

Comment:
At some point now that we've reached three items, I think it would make sense to move this logic into `KnownFunction`, more like we do for `KnownClass` and `KnownInstance`.